src/internal_rules.jl

```diff
@@ -844,18 +840,31 @@ function EnzymeRules.forward(
     end
 end
 
-function EnzymeRules.augmented_primal(
-    config,
-    func::Const{typeof(cholesky)},
-    RT::Type,
-    A::Annotation{<:Union{Matrix,LinearAlgebra.RealHermSym{<:Real,<:Matrix}}};
-    kwargs...)
-    fact = if EnzymeRules.needs_primal(config)
+function _ldiv_Cholesky_forward!(L, U, B, dL, dU, dB)
+    ldiv!(L, B)
+    mul!(dB, dL, B, -1, 1)
+    ldiv!(L, dB)
+    ldiv!(U, B)
+    mul!(dB, dU, B, -1, 1)
+    ldiv!(U, dB)
+    return B, dB
+end
```

Comment:
This is necessary to support batch activities:
```suggestion
function _ldiv_Cholesky_forward!(L, U, B, dL, dU, dBs)
    ldiv!(L, B)
    for dB in dBs
        mul!(dB, dL, B, -1, 1)
        ldiv!(L, dB)
    end
    ldiv!(U, B)
    for dB in dBs
        mul!(dB, dU, B, -1, 1)
        ldiv!(U, dB)
    end
    return B, dBs
end
```